revolv/static/js/base/project.js

```diff
@@ -0,0 +1,68 @@
+$(document).ready(function () {
+	drawPercentagePartialCircle()	
+
+});
+
+var drawPercentagePartialCircle = function() {
+    var radius = 100,
+        padding = 10,
+        radians = 2 * Math.PI;
+
+    if ($(document).width() > 200 && $(document).width() < 800) {
+        radius = 48;
+        //radius = $(document).width()*0.16;
+    }
+
+    var dimension = (2 * radius) + (2 * padding);
+
+    var svg = d3.select(".internal-graphics-container")
```

Comment:
I think that the target class should be an argument to this function instead of a hardcoded string, so that if we change the class name in the html we don't have to change it here also